qiita_db/commands.py

```diff
@@ -189,20 +193,20 @@ def load_prep_template_from_cmd(prep_temp_path, study_id, data_type):
         prep_temp, qdb.study.Study(study_id), data_type)
 
 
-def load_parameters_from_cmd(name, fp, table):
+def load_parameters_from_cmd(name, fp, cmd_id):
     """Add a new parameters entry on `table`
 
     Parameters
     ----------
     fp : str
         The filepath to the parameters file
-    table : str
-        The name of the table to add the parameters
+    cmd_id : int
```

Comment:
@ElDeveloper you're correct - we are going to be using objects. However, this is commands.py, which is targetting the CLI, so the only way to provide the objects are by the id.